event/listener.php

```diff
@@ -377,8 +377,8 @@ public function modify_topic_last_post_time($event, $eventname)
 
 		if ($row[$data_flag . '_last_post_id'])
 		{
-			$data_row['LAST_POST_TIME'] = $this->user->format_date(!is_null($row['post_created']) ? (int) $row['post_created'] : (int) $row[$data_flag . '_last_post_time']);
-			$data_row['LAST_POST_TIME_RFC3339'] = gmdate(DATE_RFC3339, !is_null($row['post_created']) ? (int) $row['post_created'] : (int) $row[$data_flag . '_last_post_time']);
+			$data_row['LAST_POST_TIME'] = $this->user->format_date(isset($row['post_created']) ? (int) $row['post_created'] : (int) $row[$data_flag . '_last_post_time']);
+			$data_row['LAST_POST_TIME_RFC3339'] = gmdate(DATE_RFC3339, isset($row['post_created']) ? (int) $row['post_created'] : (int) $row[$data_flag . '_last_post_time']);
```

Comment:
`post_created` is 0 by default, so `isset($row['post_created'])` will return true and hence will display wrong post time (like `Jan 01 1970`).
So it would be better to test something like `(isset($row['post_created']) && (int) $row['post_created'] > 0)`.

Reply:
Okay, I will change this.